.github/workflows/crucible-ci.yaml

```diff
@@ -224,15 +228,14 @@ jobs:
         scenarios: "${{ matrix.benchmark }}"
         userenvs: "${{ matrix.userenv }}"
         ci_param_mode: "${{ matrix.param_mode }}"
-        run_samples: 1
-        repeat_runs: "no"
```

Comment:
should we replicate this to all jobs ?

Reply:
Yes, that is the plan.  After finishing this PR I was going to work on rolling out some updates like this to other repos.

Reply:
ack